backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc

```diff
@@ -12,13 +12,19 @@ Because all taken backup remains until expired, also check the time to live (TTL
 
 
 You can back up applications by creating a `Backup` custom resource (CR). For more information, see xref:../../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-creating-backup-cr.adoc#oadp-creating-backup-cr-doc[Creating a Backup CR].
+The following are the different backup types for a `Backup` CR:
 
 ifdef::openshift-rosa,openshift-rosa-hcp[]
 The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
 endif::openshift-rosa,openshift-rosa-hcp[]
 
 ifndef::openshift-rosa,openshift-rosa-hcp[]
 * The `Backup` CR creates backup files for Kubernetes resources and internal images on S3 object storage.
+
+* If you use Velero's snapshot feature to back up a persistent volume data, only snapshot related information is stored in the s3 bucket along with the Openshift object data. 
++
+If the underlying storage or the backup bucket are part of the same cluster, then the data might be lost in case of disaster.
```

Comment:
This statement regarding if the s3 storage is hosted on the cluster itself is true for ANY type of backup and should be broken out into it's own note.  Not directly under snapshots.  @shubham-pampattiwar WDYT?

Reply:
correct, separate note sounds good.